src/MagnumPlugins/AssimpImporter/Test/AssimpImporterTest.cpp

```diff
@@ -287,6 +349,690 @@ void AssimpImporterTest::openDataFailed() {
+
+        /* Scaling, linearly interpolated, sharing keys with translation */
+        CORRADE_COMPARE(animation->trackType(2), AnimationTrackType::Vector3);
+        CORRADE_COMPARE(animation->trackResultType(2), AnimationTrackType::Vector3);
+        CORRADE_COMPARE(animation->trackTargetType(2), AnimationTrackTargetType::Scaling3D);
+        CORRADE_COMPARE(animation->trackTarget(2), 2);
+        Animation::TrackView<const Float, const Vector3> scaling = animation->track<Vector3>(2);
+        CORRADE_COMPARE(scaling.interpolation(), Animation::Interpolation::Linear);
+        CORRADE_COMPARE(scaling.before(), Animation::Extrapolation::Constant);
+        CORRADE_COMPARE(scaling.after(), Animation::Extrapolation::Constant);
+        constexpr Vector3 scalingData[]{
+            Vector3{1.0f},
+            Vector3::zScale(5.0f),
+            Vector3::zScale(6.0f),
+            Vector3(1.0f),
+        };
+        CORRADE_COMPARE_AS(scaling.keys(), (Containers::StridedArrayView1D<const Float>{translationScalingKeys}), TestSuite::Compare::Container);
+        CORRADE_COMPARE_AS(scaling.values(), (Containers::StridedArrayView1D<const Vector3>{scalingData}), TestSuite::Compare::Container);
+        CORRADE_COMPARE(scaling.at(1.5f), Vector3::zScale(5.2f));
+    }
+}
+
+void AssimpImporterTest::animationGltfNoScene() {
+    /* Assimp refuses to import glTF animations if the file
+       has no scenes. This reuses the TinyGltfImporter test
+       files, not the corrected ones used by other tests. */
+    Containers::Pointer<AbstractImporter> importer = _manager.instantiate("AssimpImporter");
+    CORRADE_VERIFY(importer->openFile(Utility::Directory::join(TINYGLTFIMPORTER_TEST_DIR,
+        "animation.gltf")));
+
+    CORRADE_COMPARE(importer->animationCount(), 0);
+}
+
+void AssimpImporterTest::animationGltfTicksPerSecondPatching() {
+    if(!supportsAnimation(".gltf"))
+        CORRADE_SKIP("glTF 2 animation is not supported with the current version of Assimp");
+
+    /* This was fixed right after 5.0.0, but 5.0.1 only selected compilation
+       fixes and didn't bump the minor version. Boldly assuming the next
+       minor version will have fixes from 2019. */
+    const unsigned int version = aiGetVersionMajor()*100 + aiGetVersionMinor();
+    const bool hasInvalidTicksPerSecond = version > 500;
+    if(!hasInvalidTicksPerSecond)
+        CORRADE_SKIP("Current version of assimp correctly sets glTF ticks per second.");
+
+    Containers::Pointer<AbstractImporter> importer = _manager.instantiate("AssimpImporter");
+    CORRADE_VERIFY(importer->openFile(Utility::Directory::join(ASSIMPIMPORTER_TEST_DIR,
+        "animation.gltf")));
+
+    std::ostringstream out;
+    {
+        Warning redirectWarning{&out};
+        CORRADE_VERIFY(importer->animation(1));
+    }
+    CORRADE_VERIFY(out.str().find(" ticks per second is incorrect for glTF, patching to 1000\n")
+        != std::string::npos);
```

Comment:
```suggestion
    CORRADE_VERIFY(Containers::StringView{out.str()}.contains(" ticks per second is incorrect for glTF, patching to 1000\n"));
```

Probably needs a `#include <Corrade/Containers/StringStl.h>`. (And I should finally add "string contains" comparators for this kind of tests.)

Reply:
This is much nicer, good to know